commands/bitcount.md

```diff
@@ -10,19 +10,30 @@ byte, -2 is the penultimate, and so forth.
 
 Non-existent keys are treated as empty strings, so the command will return zero.
 
+By default the additional arguments _start_ and _end_ are byte index. We can use
```

Comment:
The guideline for redis-doc is one sentence per line and no specific max length.

Reply:
I do the same as the file. But still fix it.